sktime/forecasting/model_selection/_tune.py

```diff
@@ -1913,7 +1913,10 @@ def _run_search(self, y, X, cv, scoring, scoring_name, sampler):
         for (
             param_grid_dict
         ) in self._param_grid:  # Assuming self._param_grid is now a list of dicts
-            study = optuna.create_study(direction="minimize", sampler=sampler)
+            scoring_direction = (
+                "minimize" if scoring.get_tag("lower_is_better") else "maximize"
+            )
+            study = optuna.create_study(direction=scoring_direction, sampler=sampler)
```

Comment:
could you kindly check this line? The "from" changed after merging the "scoring direction". I tried to merge sensibly.

Reply:
Looks good to me